README.md

```diff
@@ -252,10 +268,10 @@ Create the `~/.sktmrc` file telling sktm how to access Jenkins:
     EOF
 
 Clone sktm with Git, or simply download and unpack the latest "master" branch
-archive. After that, you will be able to run the `sktm.py` executable
-directly. The examples below assume you run it while being in the source root
-directory, but any directory will do as long as the path to `sktm.py` is
-correct.
+archive. installs `sktm` by `pip install .`. After that, you will be able to
+run the `sktm` executable directly. The examples below assume you run it while
+being in the source root directory, but any directory will do as long as the
```

Comment:
This doesn't matter as long as the installation completed successfully, so mentioning it is confusing. Can you remove this part? Actually, the whole paragraph can be removed since we have the installation section.